diff --git a/node_modules/@dcloudio/uni-cli-shared/dist/vite/plugins/jsonJs.js b/node_modules/@dcloudio/uni-cli-shared/dist/vite/plugins/jsonJs.js
index 3de0e44..5480715 100644
--- a/node_modules/@dcloudio/uni-cli-shared/dist/vite/plugins/jsonJs.js
+++ b/node_modules/@dcloudio/uni-cli-shared/dist/vite/plugins/jsonJs.js
@@ -45,7 +45,29 @@ function createDefineJsonJsPlugin(name) {
             if (!opts.filter(id)) {
                 return;
             }
-            return fs_1.default.readFileSync(jsonPath, 'utf8');
+
+
+            var pagesJson = fs_1.default.readFileSync(jsonPath, 'utf8')
+            const pagesJsonJsFileName = name.split('.')[0] + '.js'
+            const pagesJsonJsPath = path_1.default.join(
+                process.env.UNI_INPUT_DIR,
+                pagesJsonJsFileName
+            )
+
+            if (fs_1.default.existsSync(pagesJsonJsPath)) {
+                delete require.cache[pagesJsonJsPath]
+                const pagesJsonJsFn = require(pagesJsonJsPath)
+                if (typeof pagesJsonJsFn === 'function') {
+                    pagesJson = pagesJsonJsFn(pagesJson)
+                    if (!pagesJson) {
+                        console.error(`${pagesJsonJsFileName}  必须返回一个 json 对象`)
+                    }
+                } else {
+                    console.error(`${pagesJsonJsFileName} 必须导出 function`)
+                }
+            }
+
+            return pagesJson
         };
         return plugin;
     };
