diff --git a/node_modules/@dcloudio/uni-cloud/lib/uni.plugin.js b/node_modules/@dcloudio/uni-cloud/lib/uni.plugin.js
index ccea4b5..92bdeac 100644
--- a/node_modules/@dcloudio/uni-cloud/lib/uni.plugin.js
+++ b/node_modules/@dcloudio/uni-cloud/lib/uni.plugin.js
@@ -44,9 +44,9 @@ const UniCloudPlugin = {
     },
     closeBundle() {
         if (process.env.UNI_PLATFORM === 'h5' && !process.env.UNI_SSR_CLIENT) {
-            console.log();
-            console.log('欢迎将H5站部署到uniCloud前端网页托管平台，高速、免费、安全、省心，详见：');
-            console.log('https://uniapp.dcloud.io/uniCloud/hosting');
+            // console.log();
+            // console.log('欢迎将H5站部署到uniCloud前端网页托管平台，高速、免费、安全、省心，详见：');
+            // console.log('https://uniapp.dcloud.io/uniCloud/hosting');
         }
     },
 };
