diff --git a/node_modules/@unocss/preset-mini/dist/chunks/default.cjs b/node_modules/@unocss/preset-mini/dist/chunks/default.cjs
index c1e8a05..a79a785 100644
--- a/node_modules/@unocss/preset-mini/dist/chunks/default.cjs
+++ b/node_modules/@unocss/preset-mini/dist/chunks/default.cjs
@@ -108,22 +108,25 @@ const lineWidth = {
   DEFAULT: "1px",
   none: "0px"
 };
-const spacing = {
-  "DEFAULT": "1rem",
-  "none": "0rem",
-  "xs": "0.75rem",
-  "sm": "0.875rem",
-  "lg": "1.125rem",
-  "xl": "1.25rem",
-  "2xl": "1.5rem",
-  "3xl": "1.875rem",
-  "4xl": "2.25rem",
-  "5xl": "3rem",
-  "6xl": "3.75rem",
-  "7xl": "4.5rem",
-  "8xl": "6rem",
-  "9xl": "8rem"
-};
+const spacing = [
+  "none",
+  "xs",
+  "sm",
+  "DEFAULT",
+  "lg",
+  "xl",
+  "2xl",
+  "3xl",
+  "4xl",
+  "5xl",
+  "6xl",
+  "7xl",
+  "8xl",
+  "9xl",
+].reduce((o, k, i) => {
+  o[k] = (i * 4/3).toFixed(2) + "vw";
+  return o
+}, {});
 const duration = {
   DEFAULT: "150ms",
   none: "0ms"
diff --git a/node_modules/@unocss/preset-mini/dist/chunks/default.mjs b/node_modules/@unocss/preset-mini/dist/chunks/default.mjs
index 195bea6..f4e84bf 100644
--- a/node_modules/@unocss/preset-mini/dist/chunks/default.mjs
+++ b/node_modules/@unocss/preset-mini/dist/chunks/default.mjs
@@ -106,22 +106,25 @@ const lineWidth = {
   DEFAULT: "1px",
   none: "0px"
 };
-const spacing = {
-  "DEFAULT": "1rem",
-  "none": "0rem",
-  "xs": "0.75rem",
-  "sm": "0.875rem",
-  "lg": "1.125rem",
-  "xl": "1.25rem",
-  "2xl": "1.5rem",
-  "3xl": "1.875rem",
-  "4xl": "2.25rem",
-  "5xl": "3rem",
-  "6xl": "3.75rem",
-  "7xl": "4.5rem",
-  "8xl": "6rem",
-  "9xl": "8rem"
-};
+const spacing = [
+  "none",
+  "xs",
+  "sm",
+  "DEFAULT",
+  "lg",
+  "xl",
+  "2xl",
+  "3xl",
+  "4xl",
+  "5xl",
+  "6xl",
+  "7xl",
+  "8xl",
+  "9xl",
+].reduce((o, k, i) => {
+  o[k] = (i * 4/3).toFixed(2) + "vw";
+  return o
+}, {});
 const duration = {
   DEFAULT: "150ms",
   none: "0ms"
diff --git a/node_modules/@unocss/preset-mini/dist/chunks/utilities.cjs b/node_modules/@unocss/preset-mini/dist/chunks/utilities.cjs
index 6d93a2a..3ab0654 100644
--- a/node_modules/@unocss/preset-mini/dist/chunks/utilities.cjs
+++ b/node_modules/@unocss/preset-mini/dist/chunks/utilities.cjs
@@ -389,7 +389,7 @@ function rem(str) {
   const [, n, unit] = match;
   const num = parseFloat(n);
   if (!Number.isNaN(num))
-    return unit ? `${round(num)}${unit}` : `${round(num / 4)}rem`;
+    return unit ? `${round(num)}${unit}` : `${round(num)}vw`;
 }
 function px(str) {
   if (str.match(unitOnlyRE))
diff --git a/node_modules/@unocss/preset-mini/dist/chunks/utilities.mjs b/node_modules/@unocss/preset-mini/dist/chunks/utilities.mjs
index 052ab94..93b6e03 100644
--- a/node_modules/@unocss/preset-mini/dist/chunks/utilities.mjs
+++ b/node_modules/@unocss/preset-mini/dist/chunks/utilities.mjs
@@ -387,7 +387,7 @@ function rem(str) {
   const [, n, unit] = match;
   const num = parseFloat(n);
   if (!Number.isNaN(num))
-    return unit ? `${round(num)}${unit}` : `${round(num / 4)}rem`;
+    return unit ? `${round(num)}${unit}` : `${round(num)}vw`;
 }
 function px(str) {
   if (str.match(unitOnlyRE))
