diff --git a/node_modules/@unocss/vite/dist/index.cjs b/node_modules/@unocss/vite/dist/index.cjs
index ada5876..5e1e2f3 100644
--- a/node_modules/@unocss/vite/dist/index.cjs
+++ b/node_modules/@unocss/vite/dist/index.cjs
@@ -162,6 +162,7 @@ function GlobalModeBuildPlugin({ uno, ready, extract, tokens, modules, filter })
       enforce: "pre",
       buildStart() {
         tasks = [];
+        if(!process.env.UNI_PLATFORM.startsWith('app'))
         vfsLayerMap.clear();
       },
       transform(code, id) {
diff --git a/node_modules/@unocss/vite/dist/index.mjs b/node_modules/@unocss/vite/dist/index.mjs
index 30eadfd..12bfef6 100644
--- a/node_modules/@unocss/vite/dist/index.mjs
+++ b/node_modules/@unocss/vite/dist/index.mjs
@@ -153,6 +153,7 @@ function GlobalModeBuildPlugin({ uno, ready, extract, tokens, modules, filter })
       enforce: "pre",
       buildStart() {
         tasks = [];
+        if(!process.env.UNI_PLATFORM.startsWith('app'))
         vfsLayerMap.clear();
       },
       transform(code, id) {
