diff --git a/node_modules/lodash-es/_root.js b/node_modules/lodash-es/_root.js
index 281f812..ee54e28 100644
--- a/node_modules/lodash-es/_root.js
+++ b/node_modules/lodash-es/_root.js
@@ -3,7 +3,8 @@ import freeGlobal from './_freeGlobal.js';
 /** Detect free variable `self`. */
 var freeSelf = typeof self == 'object' && self && self.Object === Object && self;
 
+
 /** Used as a reference to the global object. */
-var root = freeGlobal || freeSelf || Function('return this')();
+var root = freeGlobal || freeSelf || global || Function('return this')();
 
 export default root;
